os/android/drmhwctwo.cpp

```diff
@@ -466,6 +466,13 @@ HWC2::Error DrmHwcTwo::HwcDisplay::PresentDisplay(int32_t *retire_fence) {
   for (std::pair<const uint32_t, DrmHwcTwo::HwcLayer *> &l : z_map) {
     layers.emplace_back(l.second->GetLayer());
   }
+  // incase cursor comes at lowest z-order, move it to higest z-order
+  auto l = z_map.begin();
```

Comment:
instead of this we can do something similar to what we do with client layer here i.e. if cursor layer is in use z_order is the last layer z_order + 1 ?